src/ruby/SphericalCoordinates.i

```diff
@@ -77,17 +98,25 @@ namespace gz
       public: gz::math::Vector3<double> SphericalFromLocalPosition(
                   const gz::math::Vector3<double> &_xyz) const;
 
+      public: std::optional<gz::math::CoordinateVector3>
+              SphericalFromLocalPosition(
+                  const gz::math::CoordinateVector3 &_xyz) const;
+
       public: gz::math::Vector3<double> GlobalFromLocalVelocity(
                   const gz::math::Vector3<double> &_xyz) const;
 
+      public: std::optional<gz::math::CoordinateVector3>
+              GlobalFromLocalVelocity(
+                  const gz::math::CoordinateVector3 &_xyz) const;
+
       public: static SurfaceType Convert(const std::string &_str);
 
       public: static std::string Convert(SurfaceType _type);
 
-      public: static double Distance(const gz::math::Angle &_latA,
```

Comment:
SphericalCoordinates.i file was present before this PR, but it wasn't built. I added it to CMakeLists and this line popped up as wrong code as the original method was renamed some time ago. So no API will be broken because SphericalCoordinates.i was not a part of API :)